src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx

```diff
@@ -95,7 +95,10 @@ export const MissionCTA = ({ CTAs }: MissionCtaProps) => {
                   </SoraTypography>
                 </CTAExplanationBox>
                 {isMobile ? undefined : (
-                  <IconButtonPrimary onClick={handleClick}>
+                  <IconButtonPrimary
+                    onClick={handleClick}
+                    sx={{ backgroundColor: theme.palette.primary.main }}
```

Comment:
Would prefer you to change the styled component instead to keep the maintainability easy